soltesz-lab-to-nwb/treadmillbehaviordatainterface.py

```diff
@@ -0,0 +1,44 @@
+"""Authors: Cesar Echavarria."""
+from pynwb import NWBFile, TimeSeries
+from nwb_conversion_tools.basedatainterface import BaseDataInterface
+from nwb_conversion_tools.utils.json_schema import (
+    get_base_schema,
+    get_schema_from_hdmf_class,
+)
+from nwb_conversion_tools.utils.common_writer_tools import check_module
+from hdmf.backends.hdf5.h5_utils import H5DataIO
+
+import numpy as np
+
+
+class TreadmillBehaviorDataInterface(BaseDataInterface):
+    """Conversion class for running speed behavioral data."""
+
+    @classmethod
+    def get_source_schema(cls):
+        """Compile input schemas from each of the data interface classes."""
+        return dict(
+            required=["file_path"], properties=dict(file_path=dict(type="string"))
+        )
```

Comment:
Instead of defining `get_source_schema`, I would recommend defining `__init__` and then having this function automatically derived from it [here](https://github.com/catalystneuro/nwb-conversion-tools/blob/1d127d41c7b8036b1c5a784f54c122099b47568c/nwb_conversion_tools/basedatainterface.py#L10-L11)